src/components/Navbar/index.tsx

```diff
@@ -1,5 +1,144 @@
+                </Link>
+                <Box sx={{ display: 'flex', flexDirection: 'row', paddingRight: '8px' }}>
+                  <Link href="/#about" className={s.navItem}>
+                    About
+                  </Link>
+                  <Link href="/#faq" className={s.navItem}>
+                    FAQ
+                  </Link>
+                  <Link href="/#companies" className={s.navItem}>
+                    Companies
+                  </Link>
+                  <Link href="https://acmurl.com/space-registration" className={s.navItem} target="_blank">
+                    Registration
+                  </Link>
+                </Box>
+              </Box>
+            </Container>
+          </AppBar>
+        </ThemeProvider>
+      </nav>
+    );
+  } else {
+    return (
+      <nav>
+        <ThemeProvider theme={theme}>
+          <AppBar color="secondary" sx={{ boxShadow: 'none' }}>
+            <Container maxWidth="xl">
+              <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between' }}>
+                <Link href="/">
+                  <img src="asset/nav_spaceship.svg" className={s.ship} />
+                </Link>
+                <IconButton
+                  size="large"
+                  edge="start"
+                  color="primary"
+                  aria-label="open drawer"
+                  sx={{ mr: 2 }}
+                  onClick={handleMobileOpen}
+                >
+                  <MenuIcon />
+                </IconButton>
+              </Box>
+            </Container>
+          </AppBar>
+          <Dialog
+            fullScreen
+            open={mobileOpen}
+            onClose={handleMobileClose}
+            PaperProps={{
+              style: {
+                backgroundColor: '#110F47',
+              },
+            }}
+          >
+            <Toolbar sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between' }}>
+              <Link href="/" onClick={handleMobileClose}>
+                <img src="asset/nav_spaceship.svg" className={s.ship} />
+              </Link>
+              <IconButton edge="start" color="primary" onClick={handleMobileClose} aria-label="close">
+                <CloseIcon />
+              </IconButton>
+            </Toolbar>
+            <Link href="/#about" className={s.navItem} onClick={handleMobileClose}>
+              About
+            </Link>
+            <Link href="/#faq" className={s.navItem} onClick={handleMobileClose}>
+              FAQ
+            </Link>
+            <Link href="/#companies" className={s.navItem} onClick={handleMobileClose}>
+              Companies
+            </Link>
+            <Link
+              href="https://acmurl.com/space-registration"
+              className={s.navItem}
+              onClick={handleMobileClose}
+              target="_blank"
+            >
+              Registration
+            </Link>
```

Comment:
Do you think that it might cause issues later down the line if we're defining the navbar contents twice? For example, if we add or update a navbar link, we might only update one of the copies. Maybe we could define navbar items in an array somewhere, or is the current way fine?

Or instead of an array, we could move the `<Link />` elements themselves into a component that returns a fragment or something. Then the mobile navbar's `onClick={handleMobileClose}`s could be moved to a parent element, and then `handleMobileClose` can check if `event.target` is inside a `Link` element somehow

Reply:
I originally thought that since we don't update this website much and we'd have to distinguish between an internal nav link and a link to send us to another site (since I want it to open in a new tab) that it wasn't worth it to refactor into a list.

But I ended up just adding a couple flags in the pages list and now it works well and is pretty modular 👍